source/style-guide/style/metadata.txt

```diff
@@ -0,0 +1,19 @@
+.. _metadata:
+
+=============
+Metadata
+=============
+
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
+
+Use ``.. meta::`` to add meta tags to a page. Adding meta tags aids in 
+searches both internally and via the Internet.
+
+.. code-block:: rst
+     
+   .. meta::
+      :description: read concern, local read concern, read isolation, 
+      transactions, multi-document transactions 
+
```

Comment:
We should [edit the existing page](https://www.mongodb.com/docs/meta/style-guide/markup/directives/metadata/) instead of the adding this new one.

```suggestion
.. _metadata:

=============
Metadata
=============

.. default-domain:: mongodb

Use ``.. meta::`` to add |html| meta tags to a page. Adding meta tags aids in 
searches both internally and via the Internet. You can add both keywords and descriptions.

.. code-block:: rst
     
   .. meta::
      :keywords: read concern, local read concern, read isolation, transactions, multi-document transactions
      :description: You can tune the consistency and availability of your application using write concerns and read concerns.

Add the ``.. meta::`` directive on the line below the ``.. default-domain::`` directive with a line break in between.

Keywords
----------

- Take the most concise form of the information that the page conveys and make that the target 
  keyword.
- Don't add keywords so broad that they compete with the product page.
- Don't add keywords so specific that we miss the actual behavior of our searchers.
  For example, add “mongodb atlas course” instead of “Getting Started with Atlas Data Platform”.
  
Descriptions
-------------

- Write the snippet that appears under the link in the search results.
- Make the descriptions informative (and enticing if possible).
- Emphasize the “why” for using the page.
- Use a maximum of 155 characters .
- Include target keyword and a call to action.
- Keep the description short, simple and concise. 
- Concisely explain what the page is about.
- Use unique meta descriptions for every page.

**Examples:**

.. code-block:: rst

   .. meta::
      :description: Use a language analyzer to create search keywords in your Atlas Search index that are optimized for a particular natural language.
      
.. code-block:: rst

   .. meta::
      :description: Use the character filters in an Atlas Search custom analyzer to examine text one character at a time and perform filtering operations.
 
```

Reply:
I agree that we should use the existing page because the basic information about the tag is relevant in the "markup/directives" section, but the additional info that provides guidance seems beyond the scope of the rest of the content in that section. For example, [see the content in markup/directives/code-blocks.txt](https://github.com/mongodb/docs-meta/blob/master/source/style-guide/markup/directives/code-blocks.txt).

I think we should do the following:
1) fix up the description of the directive in this PR
2) in a future task, create a new page (e.g. "Meta Tag Guidelines") that provides guidance on similar to the [Error Message Guidelines page](https://www.mongodb.com/docs/meta/style-guide/error-message-guidelines/).
3) add a link from this page to the new one